README.md

```diff
@@ -1,65 +1,89 @@
 <div align="center">
     <div>
-        <!-- Load images from raw.githubusercontent.com to enable image rendering when viewed from https://www.npmjs.com/package/@ni/eslint-config -->
-        <img src="https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg" alt="JavaScript, TypeScript, and NI logo" width="300" height="100">
+        <!-- Load images from raw.githubusercontent.com to enable image rendering when viewed from npmjs.com -->
+        <img src="https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/docs/logo.svg" alt="JavaScript, TypeScript, and NI logo" width="300" height="100">
```

Comment:
I moved the docs assets to the docs folder so these won't render until after the package is published and the assets are available at those URLs.